sky/provision/paperspace/instance.py

```diff
@@ -283,7 +290,7 @@ def query_instances(
     statuses: Dict[str, Optional[status_lib.ClusterStatus]] = {}
     for inst_id, inst in instances.items():
         status = status_map[inst['state']]
-        if non_terminated_only and status is None:
+        if non_terminated_only:
             continue
         statuses[inst_id] = status
     return statuses
```

Comment:
Since we now allow OPEN_PORTS for paperspace, we should add a `open_ports` function (can be an empty function) and expose it through `paperspace/__init__.py` to make `sky launch --ports 8000 --cloud paperspace` work.